src/igvfd/schemas/changelogs/auxiliary_set.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 11
 
+* Adjust `file_set_type` enum list to remove `variant sequencing`.
```

Comment:
Should be in a new section for schema ver 12

Reply:
fixed, my mistake.